packages/main/src/Calendar.js

```diff
@@ -153,14 +183,29 @@ class Calendar extends UI5Element {
 
 	onBeforeRendering() {
 		const oYearFormat = DateFormat.getDateInstance({ format: "y", calendarType: this._primaryCalendarType });
+		const minDateParsed = this.minDate && this.getFormat().parse(this.minDate);
+		const maxDateParsed = this.maxDate && this.getFormat().parse(this.maxDate);
+		let currentMonth = 0;
+		let currentYear = 1;
+
+		if ((this.minDate || this.maxDate) && this._timestamp && !this.isInValidRange(this._timestamp * 1000)) {
+			if (this._minDate) {
+				this.timestamp = this._minDate / 1000;
+			} else {
+				this.timestamp = (new Date(-62135596800000)).getTime() / 1000;
```

Comment:
Magic number. Please use a constant that explains what this is

Reply:
done